README.md

```diff
@@ -59,6 +59,107 @@ const formats = {
 expect.extend(matchersWithOptions({ formats }));
 ```
 
+### Verbose errors
```

Comment:
Include a code example of how to enable the verbose logging

Reply:
Good point! I've added an example now with an inline snapshot for clarity.